beacon-chain/sync/rpc_send_request.go

```diff
@@ -169,9 +169,9 @@ func SendBlobsByRangeRequest(ctx context.Context, tor blockchain.TemporalOracle,
 	}
 	defer closeStream(stream, log)
 
-	maxBlobsPerBlock := uint64(params.BeaconConfig().MaxBlobsPerBlock(req.StartSlot))
+	maxBlobsPerBlock := uint64(params.BeaconConfig().MaxBlobsPerBlock(req.StartSlot + primitives.Slot(req.Count)))
```

Comment:
can you extract req.StartSlot + req.Count to its own var ?

Reply:
reverted

Reply:
fixed in your branch

Reply:
no longer needed